pkgs/applications/emulators/yuzu/mainline.nix

```diff
@@ -131,7 +131,7 @@ stdenv.mkDerivation(finalAttrs: {
   ];
 
   # Does some handrolled SIMD
-  env.NIX_CFLAGS_COMPILE = "-msse4.1";
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.targetPlatform.sse4_2Support "-msse4.1";
```

Comment:
Why not `stdenv.hostPlatform.sse4_1Support`?

Reply:
It should actually be `hostPlatform.isx86_64` probably, because our "generic x86_64" doesn't actually advertise SSE4, which is why we need to force it here, or Yuzu won't compile.